src/main/java/org/apache/commons/pool2/impl/BaseGenericObjectPool.java

```diff
@@ -783,12 +783,21 @@ final void assertOpen() throws IllegalStateException {
      */
     final void startEvictor(final long delay) {

```

Comment:
Not specific to this PR, but IMO this name needs a scale postfix, like `delayMillis`.

Reply:
Agreed, but probably best to do separately.